great_expectations/expectations/expectation.py

```diff
@@ -2285,7 +2285,7 @@ def _validate_metric_value_between(
         else:
             below_max = True
 
-        success = above_min and below_max
+        success = bool(above_min and below_max)
```

Comment:
Is this just for readability or was it required by a linter? I think its assumed to be a bool based on the conditional no?

Reply:
Nice catch! This was an unrelated bugfix for @alexsherstinsky due to issues when the variables are `np.bool`.